### PR TITLE
Update Live reload configuration page to include Copilot

### PR DESCRIPTION
**Pull request by MarcinVaadin**
Fixes: https://github.com/vaadin/copilot-internal/issues/3838

### CODE REVIEW

**Comment by MarcinVaadin** [comment]
> Would make sense to have the live reload docs in Copilot instead and refer to that

@Artur- do you still think all of those subpages should be moved to Copilot? They are pretty generic and IMO fit in Configuration section.

I've updated pages to include information about Copilot.

File: articles/flow/configuration/live-reload/hotswap-agent.adoc [hunk 1]
```diff
@@ -45,6 +46,13 @@ In the same dialog you need to click menu:Modify options[Add VM options] to be a
 
 If you have another type of application, you'll need to pass the JVM options in the appropriate way.
 
+== Using Vaadin Copilot and Vaadin IDE Plugin
```
Comment (Artur-) [inline articles/flow/configuration/live-reload/hotswap-agent.adoc:49]: the page should start from the preferred way, not from a manual way mostly don't need

File: articles/flow/configuration/live-reload/hotswap-agent.adoc [hunk 2]
```diff
@@ -12,36 +12,50 @@ JetBrains Runtime offers hotswapping of any type of Java changes into a running
...
+
 
-== JetBrains Runtime
+=== JetBrains Runtime
 
 You can download the latest version from the https://github.com/JetBrains/JetBrainsRuntime/releases[JetBrains GitHub release page]. Be sure to select the correct architecture. Use JetBrains Runtime to execute your application, not necessarily your IDE.
 
 
-== HotswapAgent
+=== `HotSwapAgent`
```
Comment (Artur-) [inline articles/flow/configuration/live-reload/hotswap-agent.adoc:41]: I think only the logo use a capital S and everywhere else the name is written as `HotswapAgent`

Comment (Artur-) [reply articles/flow/configuration/live-reload/hotswap-agent.adoc:41]:
```suggestion
=== `HotswapAgent`
```

File: articles/flow/configuration/live-reload/hotswap-agent.adoc [hunk 3]
```diff
@@ -12,36 +12,50 @@ JetBrains Runtime offers hotswapping of any type of Java changes into a running
...
 
-You can download HotSwapAgent from the https://github.com/HotswapProjects/HotswapAgent/releases[HotswapAgent GitHub release page]. You'll need version 1.4.2 or later. 
+You can download HotSwapAgent from the https://github.com/HotswapProjects/HotswapAgent/releases[HotswapAgent GitHub release page]. You'll need version 1.4.2 or later.
```
Comment (Artur-) [inline articles/flow/configuration/live-reload/hotswap-agent.adoc:43]:
```suggestion
You can download HotswapAgent from the https://github.com/HotswapProjects/HotswapAgent/releases[HotswapAgent GitHub release page]. You'll need version 1.4.2 or later.
```

File: articles/flow/configuration/live-reload/hotswap-agent.adoc [hunk 4]
```diff
@@ -12,36 +12,50 @@ JetBrains Runtime offers hotswapping of any type of Java changes into a running
...
 
-Setting up hotswapping consists of three steps:
+
+== Using Vaadin Copilot & Vaadin IDE Plugin
+
+Depending on your project and the IDE used, Copilot can download the proper version of JDK and suggest a Vaadin plugin for your IDE.
+
+When debugging applications using *Debug using HotSwap Agent* from Vaadin IDE, the plugin checks if you're using the proper JDK, if HotSwap Agent is available, and applies additional required JVM arguments.
+
+Vaadin IDE Plugin also provides a *Compile on save* option, which can be enabled in the IDE settings. This may differ depending on your IDE.
```
Comment (Artur-) [inline articles/flow/configuration/live-reload/hotswap-agent.adoc:22]: And what does this thing that may or may not exist do, and how is it related to hotswapping and HotswapAgent?

File: articles/flow/configuration/live-reload/hotswap-agent.adoc [hunk 5]
```diff
@@ -12,36 +12,50 @@ JetBrains Runtime offers hotswapping of any type of Java changes into a running
 
 In addition to getting the Java changes into the running JVM, most frameworks have some global data that needs to be updated when a class changes -- or some caches that need to be cleared for the new class to take effect. HotswapAgent is a collection of plugins for various frameworks, which offers these kinds of global updates and cache clearing when needed.
 
-Setting up hotswapping consists of three steps:
+
+== Using Vaadin Copilot & Vaadin IDE Plugin
+
+Depending on your project and the IDE used, Copilot can download the proper version of JDK and suggest a Vaadin plugin for your IDE.
+
+When debugging applications using *Debug using HotSwap Agent* from Vaadin IDE, the plugin checks if you're using the proper JDK, if HotSwap Agent is available, and applies additional required JVM arguments.
```
Comment (Artur-) [inline articles/flow/configuration/live-reload/hotswap-agent.adoc:20]: There is nothing called "Vaadin IDE" so this and below need to be rephrased

File: articles/flow/configuration/live-reload/hotswap-agent.adoc [hunk 6]
```diff
@@ -12,36 +12,50 @@ JetBrains Runtime offers hotswapping of any type of Java changes into a running
 
 In addition to getting the Java changes into the running JVM, most frameworks have some global data that needs to be updated when a class changes -- or some caches that need to be cleared for the new class to take effect. HotswapAgent is a collection of plugins for various frameworks, which offers these kinds of global updates and cache clearing when needed.
 
-Setting up hotswapping consists of three steps:
+
+== Using Vaadin Copilot & Vaadin IDE Plugin
+
+Depending on your project and the IDE used, Copilot can download the proper version of JDK and suggest a Vaadin plugin for your IDE.
```
Comment (Artur-) [inline articles/flow/configuration/live-reload/hotswap-agent.adoc:18]: This sounds super vague, like "maybe if you have some project and some IDE, then something might do something". Can we say more like "If you are using IntelliJ and the Vaadin plugin, you can launch your application using the 'Debug using HotswapAgent' option without any additional configuration"

File: articles/flow/configuration/live-reload/hotswap-agent.adoc [hunk 7]
```diff
@@ -12,36 +12,50 @@ JetBrains Runtime offers hotswapping of any type of Java changes into a running
...
+
+== Manual Installation
+
+Setting up hotswapping manually, consists of three steps:
 
 - Install JetBrains Runtime;
 - Download HotswapAgent and install it into JetBrains Runtime; and
 - Run the application using JetBrains Runtime with additional parameters.
 
+All of the above can be done automatically using Vaadin Copilot together with Vaadin IDE Plugin.
```
Comment (Artur-) [inline articles/flow/configuration/live-reload/hotswap-agent.adoc:33]: But you don't need Vaadin Copilot for this at all?

**Review by Artur- — CHANGES_REQUESTED** [review]

File: articles/flow/configuration/live-reload/hotswap-agent.adoc (hunk 6 above)
Comment (Artur-) [reply articles/flow/configuration/live-reload/hotswap-agent.adoc:18]: If we want to mention the VS Code plugin also, then mention that also specifically

**Review by Artur- — APPROVED** [review]

**Pull request merged**